### DISCUSSION TITLE
绳网开发~开始一步一步填坑

### DISCUSSION THREAD

**Discussion by Hanabiiiiii** [share121/inter-knot#1959] [其他]
![358015126825E141E752C3B0B8961A6B](https://github.com/user-attachments/assets/9f7720bd-aa6f-4610-aee2-16730f0ea8b6)
前几天发了一篇帖子说想建一个绳网。其实这个想法早在zzz内测的时候就已经有了，奈何本人重度拖延，一直拖到现在，自从发了那篇帖子发现还有挺多人支持的，甚至还有要和我合作的推其他项目的，奈何本人太菜，没敢答应(乐)，现在是时候开始整点东西了，一直混也不是个事，奶奶滴，奥利给，淦了xdm！不就是个项目吗！慢慢学慢慢建呗！

博客不定期记录建站进展：blog.hanbaka.cn

绳网站点：[www.inter-knot.top](https://github.com/share121/inter-knot/discussions/www.inter-knot.top)  已经确认用这个域名了，感兴趣的话可以先收藏一下，不过要到真建成的那天估计有的等

以上，完毕

> 最近事一堆，压力爆炸，已经开始焦虑失眠了,淦！

**Comment by VacuolePaoo** [comment]
敢在我的地盘上打广告？

**Reply by Hanabiiiiii** [reply]
虚空广告

**Comment by lllliufeng** [comment]
好！
